src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -238,8 +246,7 @@ def test_WINNF_FT_S_SIQ_3(self):
             for channel in channels_pal[0]))
 
     # Checks for cbsd 2 (index 1)
-    self.assertChannelsContainFrequencyRange(channels_low[1], freq_range_low)
-    self.assertChannelsContainFrequencyRange(channels_high[1], freq_range_high)
+    self.assertChannelsWithinFrequencyRange(all_channels[1], freq_range_all)
```

Comment:
This does not match the test spec, since a SAS which responds with zero available channels  for C2 would pass, although the test spec requires that they give *at least* [3550,F] and [F,3650] as available.

Reply:
Yes, agreed. This is also what I was thinking while running. So what I am thnking
1. for satisfying the "at least" requirement, we shall make sure the required frequency ranges are returned
2. on top of 1, we shall allow SAS to return more than the "at least" requirement as long as the returned frequency ranges are valid.

Do you agree ?

The challenge is then how to handle the complicity in the test code logic to satisfy the above especially for 2 when there are so many combinations. My current thinking is to check if any one of the following is true:
1. SAS returned [3550,F] and [F,3650] 
2. SAS returned [3550,F+f(low)1] and [F(high)-f2,3650], where F(low)+f1 <= F(high)-f2
3. SAS returned [3550,F+f(low)1] and [F(high)-f2, f3], where f3 < 3700 for C2 or 3650 for C4
4. SAS returned [3550, 3700-f4] for C2 and [3550-3650] for C4, where 3700-f4 >= 3650

There are a lot other combinations we may have to skip. let me know your opinion. Also for C1 and C3 shall we also allow extending beyond "at least" requirements ?